pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -448,6 +461,49 @@ func (i *CommandInitOption) RunInit(_ io.Writer, parentCommand string) error {
 	return nil
 }
 
+// get kube components registry
+func (i *CommandInitOption) kubeRegistry() string {
+	registry := i.KubeImageRegistry
+	mirrorCountry := strings.ToLower(i.KubeImageMirrorCountry)
+
+	if registry != "" {
+		return registry
+	}
+
+	if registry == "" && mirrorCountry != "" {
```

Comment:
```suggestion
	if mirrorCountry != "" {
```

At this point, `registry` has been empty